test/sample_chronos_data.json

```diff
@@ -0,0 +1,8 @@
+{
+  "schedule": "R/2014-09-25T17:22:00Z/PT2M",
+  "name": "internal-test-team",
```

Comment:
Can this name be something like internal-test-team-sample-job, instead of just the team name.